server/services/homekit/lib/notifyChange.js

```diff
@@ -0,0 +1,97 @@
+const { intToRgb, rgbToHsb } = require('../../../utils/colors');
+const {
+  DEVICE_FEATURE_CATEGORIES,
+  DEVICE_FEATURE_TYPES,
+  EVENTS,
+  DEVICE_FEATURE_UNITS,
+} = require('../../../utils/constants');
+const { normalize } = require('./utils');
+const { fahrenheitToCelsius } = require('../../../utils/units');
+const { mappings } = require('./deviceMappings');
+
+/**
+ * @description Create HomeKit accessory service.
+ * @param {Object} accessories - HomeKit accessories.
+ * @param {Object} event - Gladys event to forward to HomeKit.
+ * @returns {undefined}
+ * @example
+ * notifyChange(accessories, event)
+ */
+function notifyChange(accessories, event) {
```

Comment:
Just a small feedback: Sending every device state change to HomeKit is super heavy, some device send many state changes per second, and usually the only relevant information is the last value.

In the Alexa/Google Home integration, we send only 1 state per 5 seconds maximum per device. That's what Google recommended. Only the "last" value is relevant. 

See: https://github.com/GladysAssistant/Gladys/blob/master/server/lib/gateway/gateway.forwardDeviceStateToAlexa.js#L109

and: https://github.com/GladysAssistant/Gladys/blob/master/server/lib/gateway/index.js#L55

Reply:
I improved that as asked (see https://community.gladysassistant.com/t/integration-homekit-et-siri/7493/48)